app/controllers/events/view/tickets/speakers/list.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
```

Comment:
@sumedh123 The problem here is that this is not the correct controller, There is no route called `events/view/tickets/speakers/list`, you need to create a controller for `/events/view/speakers/list` instead. Did not notice it at all 😅

Reply:
@sumedh123 Can you please make these changes, so that we can merge the PR?

Reply:
@geekyd Working.

Reply:
@sumedh123 Can you please check why tests are failing?

Reply:
@geekyd Checking. Will push again.

Reply:
@geekyd I think the tests are failing since open-event-api is down. I am also not able to run the tests locally.